rqd/rqd/rqmachine.py

```diff
@@ -375,6 +375,48 @@ def getTempPath(self):
             return "/mcp/"
         return '%s/' % tempfile.gettempdir()
 
+    @rqd.rqutil.Memoize
+    def __getWindowsCpuCount(self):
+        """Counts the actual number of physical CPUs on Windows"""
+
+        import ctypes
```

Comment:
Do we need to import this here or can it be at the top of the file?

Reply:
Obsoleted by recent change.